### PR TITLE
Add user related parameters

### LINKED ISSUE #8: Add user to the schema
## Summary 
- Need to add user parameters to the schema

### PR DESCRIPTION
**Pull request by xvepkj**
### Purpose of this change

Needs to be done

### Change log

Changes

Describe changes

### Verifying this change

- Does this change require testing? yes/**no**
- How is this tested? **not applicable**/automated tests/manual testing

### Documentation

- Does this feature require documentation? yes/**no**
- If yes, how is it documented? **not applicable**/docs/inline

***

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -489,9 +538,20 @@
...
+  "userName" : string,
+  "email" : string,
+  "password" : string
+}
+```
+
 ### Event
 ```
 {
+  "userId": string,
```
Comment (gnaagar) [inline docs/api.md]: I think we don't need `userId` for every non-user schema.

**Review by gnaagar — CHANGES_REQUESTED** [review]

File: docs/api.md (hunk 1 above)
Comment (xvepkj) [reply docs/api.md]: can remove for active template as it contains templateId, where else you think we can remove it?

Comment (gnaagar) [reply docs/api.md]: We can say that all routes related to events, templates, tags, stats are secured and scoped to the authenticated user.

Comment (xvepkj) [reply docs/api.md]: Okay..., have removed it from the schemas.

File: docs/api.md [hunk 2]
```diff
@@ -278,15 +278,18 @@
...
-      "name": string,
-      "startTime": string,
-      "endTime": string,
-      "trackingType": string in ["UNTRACKED", "TRACKED", "TIME_TRACKED"]
-      "tagId": string OR null
-    }
-  ]
+  {
+    "name": string
+    [
```
Comment (gnaagar) [inline docs/api.md]: add "events" key

**Review by gnaagar — CHANGES_REQUESTED** [review]

**Review by gnaagar — APPROVED** [review]
use appropriate commit messages

**Pull request merged**